### PR TITLE
Ch add all test files

### PR DESCRIPTION
**Pull request by e-liyai**
- add travis integration
- mock vscode lib
- add add lib command tests
- add test coverage
- update jest-cli package

### CODE REVIEW

File: lib/pivotaly/createPTStatusBarItem.spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const {createPTStatusBarItem} = require('./createPTStatusBarItem')
+
+describe('#createPTStatusBarItem', () => {
+  test('test create status bar object', () => {
+    expect(typeof createPTStatusBarItem()).toBe('object')
```
Comment (brayovsky) [inline lib/pivotaly/createPTStatusBarItem.spec.js]: I think this has undesired side effects. Can we mock vscode first?

**Comment by brayovsky** [comment]
Can we add codacy to checks?

File: lib/pivotaly/createPTStatusBarItem.spec.js (hunk 1 above)
Comment (e-liyai) [reply lib/pivotaly/createPTStatusBarItem.spec.js]: @brayovsky all tests are using a mocked `vscode`. I don't need to explicitly call the mock at the top of the file

**Comment by e-liyai** [comment]
@brayovsky 👌

File: lib/pivotaly/createPTStatusBarItem.spec.js (hunk 1 above)
Comment (brayovsky) [reply lib/pivotaly/createPTStatusBarItem.spec.js]: @e-liyai 👍

**Pull request merged**